runtime-modules/project-token/Cargo.toml

```diff
@@ -1,33 +1,35 @@
 [package]
 name = "pallet-project-token"
-version = '0.1.0'
+version = '5.0.0'
```

Comment:
That makes sense of course, but since we never publish anything to a remote crates registry or use our crates in other projects, it doesn't really make a difference :) But I will commit suggestion

Reply:
updated in 27f693078d3a90bd676a3f774f4b5ca837382b99